integration-tests/src/test/resources/performance-features/continuously-create-bridges.feature

```diff
@@ -0,0 +1,130 @@
+                  handler:
+                    body:
+                      json:
+                        query: .id
+                        toVar: bridgeId
+            - create-poll:
+              # Wait until the Bridge is in either in ready or failed state
+              - httpRequest:
+                  endpoint: manager
+                  GET: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                    cache-control: no-cache
+                  handler:
+                    body:
+                      json:
+                        query: .status
+                        toVar: bridgeStatus
+              - breakSequence:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: ready
+                  onBreak:
+                    newSequence:
+                      sequence: delete
+                      forceSameIndex: true
+              - fail:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: failed
+                  message: "Bridge creation failed"
+              - thinkTime:
+                  duration: 5s
+              - restartSequence
+            - delete:
+              # Delete Bridge
+              - httpRequest:
+                  endpoint: manager
+                  DELETE: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    authorization: Bearer ${accessToken}
+            - delete-poll:
+              # Wait until the Bridge is in either deleted or in failed state
+              - httpRequest:
+                  endpoint: manager
+                  GET: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                    cache-control: no-cache
+                  handler:
+                    status:
+                      store:
+                        toVar: reponseStatus
+                    body:
+                      json:
+                        query: .status
+                        toVar: bridgeStatus
+              - breakSequence:
+                  intCondition:
+                    fromVar: reponseStatus
+                    equalTo: 404
+              - fail:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: failed
+                  message: "Bridge deletion failed"
+              - thinkTime:
+                  duration: 5s
+              - restartSequence
+      """
+    Then run benchmark "rhose-create-bridges" within 15 minutes
+    And the benchmark run "rhose-create-bridges" was executed successfully
+    And store Manager metrics in Horreum test "continuously-create-bridges-<users>-users"
```

Comment:
why are we storing only the manager metrics and not the hyperfoil results of the benchmark?

Reply:
The only metric which I check in this scenario for now is the Bridge creation/deletion elapsed time.
Do you know any metric from Hyperfoil which would have sense to watch in here?

Reply:
not very familiar yet with all the metrics provided by hyperfoil, so it's ok to me if you think that's enough.

Reply:
It can be extended later if needed.